pkgs/applications/office/abiword/default.nix

```diff
@@ -5,15 +5,19 @@
 
 stdenv.mkDerivation rec {
   name = "abiword-${version}";
-  version = "3.0.1";
+  version = "3.0.2";
 
   src = fetchurl {
-    url = "http://www.abisource.org/downloads/abiword/${version}/source/${name}.tar.gz";
-    sha256 = "1ik591rx15nn3n1297cwykl8wvrlgj78i528id9wbidgy3xzd570";
+    url = "http://www.abisource.com/downloads/abiword/${version}/source/${name}.tar.gz";
+    sha256 = "08imry821g81apdwym3gcs4nss0l9j5blqk31j5rv602zmcd9gxg";
   };
 
   enableParallelBuilding = true;
 
+  patches = [
+    (fetchurl { url = https://gist.github.com/ylwghst/b28ef051980f41aca56df7a8affd9526/raw/; sha256 = "02p8kz02xm1197zcpzjs010mna9hxsbq5lwgxr8b7qhh9yxja7al"; })
```

Comment:
I think we'd prefer this patch be in nixpkgs, than just in your gists.

Reply:
In fact, just use two patches, linking to the original URLs you already put in the body of the issue?

Reply:
When I try to fetch this URL: https://bugzilla.abisource.com/attachment.cgi?id=5860&format=raw
Patch is fetched as 'attachment.cgi?id=5860&format=raw' which is problem due to the & in it's name.

Reply:
You can add `name` attribute but it should not be necessary to add `format`.